src/patternfly/components/NotificationBadge/notification-badge.scss

```diff
@@ -33,12 +33,18 @@
   --pf-c-notification-badge--m-attention--after--BackgroundColor: var(--pf-global--danger-color--100);
 
   // Badge count
-  --pf-c-notification-badge__count--MarginLeft: var(--pf-global--spacer--xs);
+  --pf-c-notification-badge--ColumnGap: var(--pf-global--spacer--xs);
+
+  // Child
+  --pf-c-notification-badge--child--LineHeight: var(--pf-global--LineHeight--md);
```

Comment:
If you're setting `line-height` on the font awesome icon (since `.fas` assigns `line-height: 1`), is this necessary? Both the text and SVG will have the default line height.